### ISSUE TITLE
Complete profile page.

### ISSUE THREAD

**Issue by WickedBrat1** [Cynthesize/frontend#37]
Fetch data from profile URL and use it to complete the profile page as developed in PR #35 
The backend URL is `/api/user/view/<username>` whereas on the frontend the URL is at `/users/<user-id>`
- [ ] Change userid in the frontend to username and get that username from URL to maki API call.
- [ ] Fetch data from backend and embed the data with proper conditions on the page

Make different pull requests for this.

**Comment by WickedBrat1** [comment]
@vivekpatel5996 would you like to take this?

**Comment by vivekpatel5996** [comment]
> @vivekpatel5996 would you like to take this?
Yes I would like that. but i can't see profile page is integrated with design.

**Comment by zerefwayne** [comment]
Try /user/1

**Comment by vivekpatel5996** [comment]
> Try /user/1
@zerefwayne No No I am talking about integrated UI. Sorry for not giving clear idea.
